src/blueprint/server.rs

```diff
@@ -4,10 +4,31 @@ use std::net::{AddrParseError, IpAddr};
 use derive_setters::Setters;
 use hyper::header::{HeaderName, HeaderValue};
 use hyper::HeaderMap;
+use serde::{Deserialize, Serialize};
 
 use crate::config;
 use crate::valid::{Valid, ValidationError};
 
+#[derive(Deserialize, Serialize, Clone, Debug, Setters)]
+#[serde(rename_all = "camelCase", default)]
+pub struct HttpOptions {
```

Comment:
How about we just call it `Http`? It is contextual to `server` and matches the field name `http`